### DISCUSSION TITLE
Message I use pro mode, but I actually do not?

### DISCUSSION THREAD

**Discussion by dijkr** [statamic/cms#9281] [Q&A]
Whenever I save a new item, the image won't load: Failed to load resource: the server responded with a status of `404 (Not Found)`
In the development server, the image load. The images already present in the development server will load when I deploy the server, but new entries won't load. Suddenly I got the message I'm in trial mode? Clearing the cache, or restarting the server won't fix this problem. How come? Another thing that happen is something at the console: `Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'interest-cohort'.`

![Statamic](https://github.com/statamic/cms/assets/89724934/4ef1a3b8-5a6e-4e1d-86b2-2379619ccaac)

**Comment by dijkr** [comment]
Could this happen if I set the wrong local IP-adress at the Laracel .env file? I set it to the correct one, rerun some scripts and it fixed the issue. At least for now. :-)

**Reply by christoferd** [reply]
How do we remove Trial Mode?
How can I set the CMS to "Free Version"?

**Reply by duncanmcclean** [reply]
@christoferd This discussion is pretty old at this point. I see someone has replied to your new discussion here: https://github.com/statamic/cms/discussions/10841.
